### PR TITLE
Quoted error messages

### PR DESCRIPTION
**Pull request by avishalom**
Fix old nit.

### CODE REVIEW

File: internal/http_client.go [hunk 1]
```diff
@@ -140,7 +140,7 @@ func (r *Response) CheckStatus(want int) error {
 	if msg == "" {
 		msg = string(r.Body)
 	}
-	return fmt.Errorf("http error status: %d; reason: %s", r.Status, msg)
+	return fmt.Errorf("http error status: %d; reason: %q", r.Status, msg)
```
Comment (hiranya911) [inline internal/http_client.go]: This should not be quoted. It is breaking a bunch of tests now.

File: messaging/messaging.go [hunk 2]
```diff
@@ -423,9 +423,9 @@ func (c *Client) makeSendRequest(ctx context.Context, req *fcmRequest) (string,
 	json.Unmarshal(resp.Body, &fe) // ignore any json parse errors at this level
 	msg := fcmErrorCodes[fe.Error.Status]
 	if msg == "" {
-		msg = fmt.Sprintf("server responded with an unknown error; response: %s", string(resp.Body))
+		msg = fmt.Sprintf("server responded with an unknown error; response: %q", string(resp.Body))
```
Comment (hiranya911) [inline messaging/messaging.go]: None of these should be quoted.

File: internal/http_client.go (hunk 1 above)
Comment (avishalom) [reply internal/http_client.go]: yes , sorry I  was playing around with it, i though i removed you from review

**Comment by hiranya911** [comment]
FWIW, I didn't object quoting the printed/logged error messages. Only the errors returned from an API.

**Pull request closed without merging**